users/templates/users/account_settings.html

```diff
@@ -13,7 +13,7 @@ <h2 class="mt-3 mb-3">Account settings</h2>
                                 <h3 class="mt-4 mb-4 text-secondary fs-6 border-bottom border-secondary">Account
                                     preferences</h3>
 
-                                <!-- Email address (manual, not part of form) -->
+                                <!-- Email address (manual, not part of form) ->
```

Comment:
Please revert this change. Valid html comments end with `-->`
THis change here most likely breaks html and for sure removes showing user email field.